### DISCUSSION TITLE
How to enable etcd grpc-proxy using config.yaml

### DISCUSSION THREAD

**Discussion by NBK27** [etcd-io/etcd#17057] [Q&A] [answered]
I dont see any options for configuring grpc-proxy in https://github.com/etcd-io/etcd/blob/main/etcd.conf.yml.sample

**Comment by jmhbnz** [comment, marked as answer]
Hey @NBK27 - Thanks for your question. To the best of my knowledge it's not currently possible to configure the grpc proxy using `etcd.conf` file, however you could write a custom systemd file for starting the proxy.

Refer: 
- https://groups.google.com/g/etcd-dev/c/buk_yP6wEqY/m/IgOSoNIZCAAJ
- https://github.com/etcd-io/etcd/tree/main/contrib/systemd/etcd3-multinode#write-systemd-service-file
